relay-server/src/endpoints/common.rs

```diff
@@ -44,6 +44,9 @@ pub enum BadStoreRequest {
     #[fail(display = "empty request body")]
     EmptyBody,
 
+    #[fail(display = "unsupported content type, expected {}", _0)]
```

Comment:
Yes, that might make sense. I didn't do this initially since that would require to allocate a String and I was thinking that usually the client would know, but your argument makes sense. I'll update 👍